pagopa/io-app
# fix: [IOCOM-736] Message retrieval fix for fast login

## Short description
This PR fixes a deadlock condition for fast login while loading message data in the MessageRouterScreen.

## List of changes proposed in this pull request
- `messageGetStatus`: reducer enhanced to handle the retry status
- `watchLoadMessageData`: additional code to handle the retry if needed

## How to test
Using the io-dev-api-server, make sure that fast login is enabled and try to load a message after the countdown has expired. After renewing the session, the message retrieval should resume and complete successfully.
